drush.services.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  update_helper.commands:
```

Comment:
This service name does not match the class name. Probably class `ApplyConfigurationUpdateCommands` should be renamed into something like `UpdateHelperCommands` if we want one class to handle multiple commands. Or we should rename service name and class to match single command per class approach.

I'm not sure what is best practice in case of Drush commands?